src/app/public/modules/page/fixtures/page-child.component.fixture.ts

```diff
@@ -0,0 +1,14 @@
+//#region imports
```

Comment:
FYI - we don't need to include the imports region anymore. VS code collapses these right out of the box.